### ISSUE TITLE
Unmark movie as seen results in app crash

### ISSUE THREAD

**Issue by bin101** [jellyfin/jellyfin-androidtv#4272]
### Describe the bug

1. Enable seen filter in movie library
2. Mark a movie as seen
3. Unmark the same movie
4. App crashes

### Logs

---
client: Jellyfin for Android TV
client_version: 0.18.0
client_repository: https://github.com/jellyfin/jellyfin-androidtv
type: crash_report
format: markdown
---
### Logs

***Stack Trace***: 
```log
java.lang.NullPointerException: Attempt to invoke virtual method 'void android.view.View.resetSubtreeAccessibilityStateChanged()' on a null object reference
	at android.view.ViewGroup.resetSubtreeAccessibilityStateChanged(ViewGroup.java:3880)
	at android.view.ViewRootImpl$SendWindowContentChangedAccessibilityEvent.run(ViewRootImpl.java:10214)
	at android.view.ViewRootImpl$SendWindowContentChangedAccessibilityEvent.runOrPost(ViewRootImpl.java:10233)
	at android.view.ViewRootImpl.postSendWindowContentChangedCallback(ViewRootImpl.java:9053)
	at android.view.ViewRootImpl.notifySubtreeAccessibilityStateChanged(ViewRootImpl.java:9233)
	at android.view.ViewGroup.notifySubtreeAccessibilityStateChanged(ViewGroup.java:3847)
	at android.view.ViewGroup.notifySubtreeAccessibilityStateChanged(ViewGroup.java:3847)
	at android.view.ViewGroup.notifySubtreeAccessibilityStateChanged(ViewGroup.java:3847)
	at android.view.ViewGroup.notifySubtreeAccessibilityStateChanged(ViewGroup.java:3847)
	at android.view.ViewGroup.notifySubtreeAccessibilityStateChanged(ViewGroup.java:3847)
	at android.view.ViewGroup.notifySubtreeAccessibilityStateChanged(ViewGroup.java:3847)
	at android.view.ViewGroup.notifySubtreeAccessibilityStateChanged(ViewGroup.java:3847)
	at android.view.ViewGroup.notifySubtreeAccessibilityStateChanged(ViewGroup.java:3847)
	at android.view.ViewGroup.notifySubtreeAccessibilityStateChanged(ViewGroup.java:3847)
	at android.view.ViewGroup.notifySubtreeAccessibilityStateChanged(ViewGroup.java:3847)
	at android.view.ViewGroup.notifySubtreeAccessibilityStateChanged(ViewGroup.java:3847)
	at android.view.ViewGroup.notifySubtreeAccessibilityStateChanged(ViewGroup.java:3847)
	at android.view.ViewGroup.notifySubtreeAccessibilityStateChanged(ViewGroup.java:3847)
	at android.view.View.notifyViewAccessibilityStateChangedIfNeeded(View.java:13950)
	at android.view.View.setSelected(View.java:24879)
	at android.widget.TextView.setSelected(TextView.java:11094)
	at android.view.ViewGroup.dispatchSetSelected(ViewGroup.java:4623)
	at android.view.View.setSelected(View.java:24875)
	at android.view.ViewGroup.dispatchSetSelected(ViewGroup.java:4623)
	at android.view.View.setSelected(View.java:24875)
	at android.view.ViewGroup.dispatchSetSelected(ViewGroup.java:4623)
	at android.view.View.setSelected(View.java:24875)
	at androidx.leanback.widget.BaseCardView.setSelected(BaseCardView.java:364)
	at android.view.ViewGroup.dispatchSetSelected(ViewGroup.java:4623)
	at android.view.View.setSelected(View.java:24875)
	at androidx.leanback.widget.FocusHighlightHelper$BrowseItemFocusHighlight.onItemFocused(FocusHighlightHelper.java:168)
	at androidx.leanback.widget.ItemBridgeAdapter$ChainingFocusChangeListener.onFocusChange(ItemBridgeAdapter.java:103)
	at android.view.View.onFocusChanged(View.java:8109)
	at android.view.View.clearFocusInternal(View.java:7946)
	at android.view.View.unFocus(View.java:7979)
	at android.view.ViewGroup.unFocus(ViewGroup.java:1203)
	at android.view.ViewGroup.unFocus(ViewGroup.java:1205)
	at android.view.ViewGroup.unFocus(ViewGroup.java:1205)
	at android.view.ViewGroup.unFocus(ViewGroup.java:1205)
	at android.view.ViewGroup.requestChildFocus(ViewGroup.java:863)
	at android.view.ViewGroup.requestChildFocus(ViewGroup.java:869)
	at android.view.View.handleFocusGainInternal(View.java:7771)
	at android.view.View.requestFocusNoSearch(View.java:13540)
	at android.view.View.requestFocus(View.java:13514)
	at android.view.ViewGroup.onRequestFocusInDescendants(ViewGroup.java:3374)
	at android.view.ViewGroup.requestFocus(ViewGroup.java:3323)
	at android.view.View.requestFocus(View.java:13481)
	at android.view.View.requestFocus(View.java:13423)
	at org.jellyfin.androidtv.ui.browsing.BrowseGridFragment.lambda$refreshCurrentItem$3$org-jellyfin-androidtv-ui-browsing-BrowseGridFragment(BrowseGridFragment.java:888)
	at org.jellyfin.androidtv.ui.browsing.BrowseGridFragment$$ExternalSyntheticLambda2.invoke(D8$$SyntheticClass:0)
	at org.jellyfin.androidtv.ui.itemhandling.ItemRowAdapterHelperKt$refreshItem$2.invokeSuspend(ItemRowAdapterHelper.kt:697)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:101)
	at kotlinx.coroutines.internal.LimitedDispatcher$Worker.run(LimitedDispatcher.kt:113)
	at kotlinx.coroutines.scheduling.TaskImpl.run(Tasks.kt:89)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:589)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:823)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:720)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:707)
	Suppressed: kotlinx.coroutines.internal.DiagnosticCoroutineContextException: [StandaloneCoroutine{Cancelling}@fb73070, Dispatchers.IO]

```  
***Logcat***: 
```log
12-05 09:26:50.636 E/ViewRootImpl(10161): 	at androidx.leanback.widget.ItemBridgeAdapter$ChainingFocusChangeListener.onFocusChange(ItemBridgeAdapter.java:103)
12-05 09:26:50.636 E/ViewRootImpl(10161): 	at android.view.View.onFocusChanged(View.java:8109)
12-05 09:26:50.636 E/ViewRootImpl(10161): 	at android.view.View.clearFocusInternal(View.java:7946)
12-05 09:26:50.636 E/ViewRootImpl(10161): 	at android.view.View.unFocus(View.java:7979)
12-05 09:26:50.636 E/ViewRootImpl(10161): 	at android.view.ViewGroup.unFocus(ViewGroup.java:1203)
12-05 09:26:50.636 E/ViewRootImpl(10161): 	at android.view.ViewGroup.unFocus(ViewGroup.java:1205)
12-05 09:26:50.636 E/ViewRootImpl(10161): 	at android.view.ViewGroup.unFocus(ViewGroup.java:1205)
12-05 09:26:50.636 E/ViewRootImpl(10161): 	at android.view.ViewGroup.unFocus(ViewGroup.java:1205)
12-05 09:26:50.636 E/ViewRootImpl(10161): 	at android.view.ViewGroup.requestChildFocus(ViewGroup.java:863)
12-05 09:26:50.636 E/ViewRootImpl(10161): 	at android.view.ViewGroup.requestChildFocus(ViewGroup.java:869)
12-05 09:26:50.636 E/ViewRootImpl(10161): 	at android.view.View.handleFocusGainInternal(View.java:7771)
12-05 09:26:50.636 E/ViewRootImpl(10161): 	at android.view.View.requestFocusNoSearch(View.java:13540)
12-05 09:26:50.636 E/ViewRootImpl(10161): 	at android.view.View.requestFocus(View.java:13514)
12-05 09:26:50.636 E/ViewRootImpl(10161): 	at android.view.ViewGroup.onRequestFocusInDescendants(ViewGroup.java:3374)
12-05 09:26:50.636 E/ViewRootImpl(10161): 	at android.view.ViewGroup.requestFocus(ViewGroup.java:3323)
12-05 09:26:50.636 E/ViewRootImpl(10161): 	at android.view.View.requestFocus(View.java:13481)
12-05 09:26:50.636 E/ViewRootImpl(10161): 	at android.view.View.requestFocus(View.java:13423)
12-05 09:26:50.636 E/ViewRootImpl(10161): 	at org.jellyfin.androidtv.ui.browsing.BrowseGridFragment.lambda$refreshCurrentItem$3$org-jellyfin-androidtv-ui-browsing-BrowseGridFragment(BrowseGridFragment.java:888)
12-05 09:26:50.636 E/ViewRootImpl(10161): 	at org.jellyfin.androidtv.ui.browsing.BrowseGridFragment$$ExternalSyntheticLambda2.invoke(D8$$SyntheticClass:0)
12-05 09:26:50.636 E/ViewRootImpl(10161): 	at org.jellyfin.androidtv.ui.itemhandling.ItemRowAdapterHelperKt$refreshItem$2.invokeSuspend(ItemRowAdapterHelper.kt:697)
12-05 09:26:50.637 E/ViewRootImpl(10161): 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
12-05 09:26:50.637 E/ViewRootImpl(10161): 	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:101)
12-05 09:26:50.637 E/ViewRootImpl(10161): 	at kotlinx.coroutines.internal.LimitedDispatcher$Worker.run(LimitedDispatcher.kt:113)
12-05 09:26:50.637 E/ViewRootImpl(10161): 	at kotlinx.coroutines.scheduling.TaskImpl.run(Tasks.kt:89)
12-05 09:26:50.637 E/ViewRootImpl(10161): 	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:589)
12-05 09:26:50.637 E/ViewRootImpl(10161): 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:823)
12-05 09:26:50.637 E/ViewRootImpl(10161): 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:720)
12-05 09:26:50.637 E/ViewRootImpl(10161): 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:707)
12-05 09:26:50.639 D/BrowseGridFragment(10161): row selected position 71
12-05 09:26:50.651 E/ACRA    (10161): ACRA caught a NullPointerException for org.jellyfin.androidtv
12-05 09:26:50.651 E/ACRA    (10161): java.lang.NullPointerException: Attempt to invoke virtual method 'void android.view.View.resetSubtreeAccessibilityStateChanged()' on a null object reference
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.ViewGroup.resetSubtreeAccessibilityStateChanged(ViewGroup.java:3880)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.ViewRootImpl$SendWindowContentChangedAccessibilityEvent.run(ViewRootImpl.java:10214)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.ViewRootImpl$SendWindowContentChangedAccessibilityEvent.runOrPost(ViewRootImpl.java:10233)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.ViewRootImpl.postSendWindowContentChangedCallback(ViewRootImpl.java:9053)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.ViewRootImpl.notifySubtreeAccessibilityStateChanged(ViewRootImpl.java:9233)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.ViewGroup.notifySubtreeAccessibilityStateChanged(ViewGroup.java:3847)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.ViewGroup.notifySubtreeAccessibilityStateChanged(ViewGroup.java:3847)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.ViewGroup.notifySubtreeAccessibilityStateChanged(ViewGroup.java:3847)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.ViewGroup.notifySubtreeAccessibilityStateChanged(ViewGroup.java:3847)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.ViewGroup.notifySubtreeAccessibilityStateChanged(ViewGroup.java:3847)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.ViewGroup.notifySubtreeAccessibilityStateChanged(ViewGroup.java:3847)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.ViewGroup.notifySubtreeAccessibilityStateChanged(ViewGroup.java:3847)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.ViewGroup.notifySubtreeAccessibilityStateChanged(ViewGroup.java:3847)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.ViewGroup.notifySubtreeAccessibilityStateChanged(ViewGroup.java:3847)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.ViewGroup.notifySubtreeAccessibilityStateChanged(ViewGroup.java:3847)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.ViewGroup.notifySubtreeAccessibilityStateChanged(ViewGroup.java:3847)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.ViewGroup.notifySubtreeAccessibilityStateChanged(ViewGroup.java:3847)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.ViewGroup.notifySubtreeAccessibilityStateChanged(ViewGroup.java:3847)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.View.notifyViewAccessibilityStateChangedIfNeeded(View.java:13950)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.View.setSelected(View.java:24879)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.widget.TextView.setSelected(TextView.java:11094)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.ViewGroup.dispatchSetSelected(ViewGroup.java:4623)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.View.setSelected(View.java:24875)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.ViewGroup.dispatchSetSelected(ViewGroup.java:4623)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.View.setSelected(View.java:24875)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.ViewGroup.dispatchSetSelected(ViewGroup.java:4623)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.View.setSelected(View.java:24875)
12-05 09:26:50.651 E/ACRA    (10161): 	at androidx.leanback.widget.BaseCardView.setSelected(BaseCardView.java:364)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.ViewGroup.dispatchSetSelected(ViewGroup.java:4623)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.View.setSelected(View.java:24875)
12-05 09:26:50.651 E/ACRA    (10161): 	at androidx.leanback.widget.FocusHighlightHelper$BrowseItemFocusHighlight.onItemFocused(FocusHighlightHelper.java:168)
12-05 09:26:50.651 E/ACRA    (10161): 	at androidx.leanback.widget.ItemBridgeAdapter$ChainingFocusChangeListener.onFocusChange(ItemBridgeAdapter.java:103)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.View.onFocusChanged(View.java:8109)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.View.clearFocusInternal(View.java:7946)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.View.unFocus(View.java:7979)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.ViewGroup.unFocus(ViewGroup.java:1203)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.ViewGroup.unFocus(ViewGroup.java:1205)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.ViewGroup.unFocus(ViewGroup.java:1205)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.ViewGroup.unFocus(ViewGroup.java:1205)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.ViewGroup.requestChildFocus(ViewGroup.java:863)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.ViewGroup.requestChildFocus(ViewGroup.java:869)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.View.handleFocusGainInternal(View.java:7771)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.View.requestFocusNoSearch(View.java:13540)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.View.requestFocus(View.java:13514)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.ViewGroup.onRequestFocusInDescendants(ViewGroup.java:3374)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.ViewGroup.requestFocus(ViewGroup.java:3323)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.View.requestFocus(View.java:13481)
12-05 09:26:50.651 E/ACRA    (10161): 	at android.view.View.requestFocus(View.java:13423)
12-05 09:26:50.651 E/ACRA    (10161): 	at org.jellyfin.androidtv.ui.browsing.BrowseGridFragment.lambda$refreshCurrentItem$3$org-jellyfin-androidtv-ui-browsing-BrowseGridFragment(BrowseGridFragment.java:888)
12-05 09:26:50.651 E/ACRA    (10161): 	at org.jellyfin.androidtv.ui.browsing.BrowseGridFragment$$ExternalSyntheticLambda2.invoke(D8$$SyntheticClass:0)
12-05 09:26:50.651 E/ACRA    (10161): 	at org.jellyfin.androidtv.ui.itemhandling.ItemRowAdapterHelperKt$refreshItem$2.invokeSuspend(ItemRowAdapterHelper.kt:697)
12-05 09:26:50.651 E/ACRA    (10161): 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
12-05 09:26:50.651 E/ACRA    (10161): 	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:101)
12-05 09:26:50.651 E/ACRA    (10161): 	at kotlinx.coroutines.internal.LimitedDispatcher$Worker.run(LimitedDispatcher.kt:113)
12-05 09:26:50.651 E/ACRA    (10161): 	at kotlinx.coroutines.scheduling.TaskImpl.run(Tasks.kt:89)
12-05 09:26:50.651 E/ACRA    (10161): 	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:589)
12-05 09:26:50.651 E/ACRA    (10161): 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:823)
12-05 09:26:50.651 E/ACRA    (10161): 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:720)
12-05 09:26:50.651 E/ACRA    (10161): 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:707)
12-05 09:26:50.651 E/ACRA    (10161): 	Suppressed: kotlinx.coroutines.internal.DiagnosticCoroutineContextException: [StandaloneCoroutine{Cancelling}@fb73070, Dispatchers.IO]
12-05 09:26:50.651 D/ACRA    (10161): Building report
12-05 09:26:50.653 D/ACRA    (10161): Starting collectors with priority FIRST
12-05 09:26:50.653 D/ACRA    (10161): Calling collector org.acra.collector.DropBoxCollector
12-05 09:26:50.654 D/ACRA    (10161): Calling collector org.acra.collector.LogCatCollector
12-05 09:26:50.654 D/ACRA    (10161): Calling collector org.acra.collector.StacktraceCollector
12-05 09:26:50.654 D/ACRA    (10161): Collector org.acra.collector.DropBoxCollector completed
12-05 09:26:50.654 D/ACRA    (10161): Collector org.acra.collector.StacktraceCollector completed
12-05 09:26:50.658 D/ACRA    (10161): Retrieving logcat output (buffer:default)...

```  

### App information

***App version***: `0.18.0` (`180099`)  
***Package name***: `org.jellyfin.androidtv`  
***Build***: 
```json
{"BOARD":"PH1M_WW_9972","BOOTLOADER":"01.01.241016","BRAND":"Philips","CPU_ABI":"armeabi-v7a","CPU_ABI2":"armeabi","DEVICE":"PH1M_WW_9972","DISPLAY":"STT2.230831.001","FINGERPRINT":"Philips\/PH1M_WW_9972_21\/PH1M_WW_9972:12\/STT2.230831.001\/20736:user\/release-keys","HARDWARE":"mt5896","HOST":"TWTPSA392-ub1804","ID":"STT2.230831.001","IS_DEBUGGABLE":false,"IS_EMULATOR":false,"MANUFACTURER":"TPV","MODEL":"Philips Google TV TA2","ODM_SKU":"unknown","PERMISSIONS_REVIEW_REQUIRED":true,"PRODUCT":"PH1M_WW_9972_21","RADIO":"unknown","SKU":"unknown","SOC_MANUFACTURER":"Mediatek","SOC_MODEL":"t50","SUPPORTED_32_BIT_ABIS":["armeabi-v7a","armeabi"],"SUPPORTED_64_BIT_ABIS":[],"SUPPORTED_ABIS":["armeabi-v7a","armeabi"],"TAGS":"release-keys","TIME":1729068575000,"TYPE":"user","UNKNOWN":"unknown","USER":"jenkins","VERSION":{"ACTIVE_CODENAMES":[],"BASE_OS":"","CODENAME":"REL","INCREMENTAL":"20736","MEDIA_PERFORMANCE_CLASS":0,"PREVIEW_SDK_FINGERPRINT":"REL","PREVIEW_SDK_INT":0,"RELEASE":"12","RELEASE_OR_CODENAME":"12","SDK":"31","SDK_INT":31,"SECURITY_PATCH":"2024-09-05"}}
```  
***Build config***: 
```json
{"APPLICATION_ID":"org.jellyfin.androidtv","BUILD_TYPE":"release","DEBUG":false,"DEVELOPMENT":false,"VERSION_CODE":180099,"VERSION_NAME":"0.18.0"}
```  

### Device information

***Android version***: `12`  
***Device brand***: `Philips`  
***Device product***: `PH1M_WW_9972_21`  
***Device model***: `Philips Google TV TA2`  

### Crash information

***Start time***: `2024-12-04T18:52:14.456+01:00`  
***Crash time***: `2024-12-05T09:26:51.108+01:00`  

### Application version

0.18.1

### Where did you install the app from?

Sideloaded APK

### Device information

philips 77oled908

### Android version

Android TV 12

### Jellyfin server version

10.10.3

**Comment by nielsvanvelzen** [comment]
I haven't been able to reproduce this one, although I do suspect it may be related to #4273 (perhaps even the exact same issue but just triggering a different exception). Can you share some more detail in how to reproduce?

**Comment by bin101** [comment]
I just activated the filter in the overview and then marked the film as seen and immediately afterwards as unseen using the long press action.

My guess is that the model/state/adapter loses the reference because of the filter and then the unseen action triggers the NPE

**Comment by nielsvanvelzen** [comment]
Ooh you used the long press menu, I didn't consider that. I can reproduce it now and it looks like the fix for #4273 also applies to this issue. So closing as fixed for 0.18.2, thanks!
